project/Dependencies.scala

```diff
@@ -54,7 +54,7 @@ object Dependencies {
   object Test {
     final val Test = sbt.Test
     val scalaTest = "org.scalatest" %% "scalatest" % Versions.scalaTest % "test" // Apache V2
-    val scalaTestPlusJunit = "org.scalatestplus" %% "junit-4-12" % "3.1.0.0" % "test" // Apache V2
+    val scalaTestPlusJunit = "org.scalatestplus" %% "junit-4-12" % (Versions.scalaTest + ".0") % "test" // Apache V2
```

Comment:
Interesting.  I'm guessing this extra version field is for incremental versions of ScalaTestPlus integrations that are compatible with a release of ScalaTest.

Reply:
indeed - and those scalatestplus integrations are independently versioned (so +junit can be on .4 while +scalatest might still be on .0)